src/test/java/TestStorage.java

```diff
@@ -0,0 +1,65 @@
+import Collections.CustomLinkedList;
+import Collections.CustomList;
+import Collections.List;
+import Containers.StorageLoadResult;
+import Containers.StorageSaveResult;
+import Containers.StorageType;
+import Containers.StorageUnit;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TestStorage {
+
+    @Test
+    void testFileStorageNotLinked(){
```

Comment:
this is a bad way to write tests. in case another developer would do some changes, he would run the tests it would fail, and he won't understand what went wrong

each class ideally should have it's own test

it is okay to have some repeativie code in tests